modules/ossm-about-cluster-wide-migration.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+
+// * service-mesh-docs-main/migrating/checklists/ossm-migrating-cluster-wide-assembly.adoc
+
+:_mod-docs-content-type: Concept
+[id="ossm-about-cluster-wide-migration_{context}"]
+= About cluster-wide migration
+
+Migrating from {SMProductName} {SMv2Version} to {SMProduct} 3.0 can be performed using one of the following methods:
```

Comment:
I would probably remove those methods and start with some more general overview in this chapter:

During the migration process, two cluster-wide control planes run in the same cluster while data plane namespaces are gradually migrated. One control plane is associated with the 2.6 installation and the other one is associated with the 3.0 installation. You must carefully plan out the migration steps in order to avoid possible conflicts between the two control planes. Next chapters describe required conditions for successful migration.